### DISCUSSION TITLE
recurrence event entries not getting deleted completely

### DISCUSSION THREAD

**Discussion by deepakd-iprogrammer** [wmcmahan/react-native-calendar-events#370] [Q&A]
### Environment
 System:
    OS: Linux 4.15 Ubuntu 18.04.5 LTS (Bionic Beaver)
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
    Memory: 291.79 MB / 19.35 GB
    Shell: 4.4.20 - /bin/bash
  Binaries:
    Node: 12.19.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.14.10 - /usr/local/bin/npm
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: Not Found
    Python: 2.7.17 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.4 => 0.63.4 
  npmGlobalPackages:
    *react-native*: Not Found### Steps to Reproduce

create a recurrence event using-
recurrenceRule: { frequency: "monthly", endDate: moment(days).add(23, 'hour').add(6, 'months').toISOString() }

after creating try to delete all entries.

### Expected Behavior

 All Recurrence event entries should get deleted on deleting their primary entry using removeEvent function.

### Actual Behavior

upon creating a recurrent event only a single entry is created for all events but when we try to delete this entry using removeEvent function, only a single entry is getting deleted.

**Comment by wmcmahan** [comment]
To remove all events in a recurring series, use the `futureEvents` option:
```javascript
removeEvent(eventID, { futureEvents: true })
```

**Comment by deepakd-iprogrammer** [comment]
thanks.

**Comment by gkasireddy202** [comment]
How to remove all events after the completion of an event like end date?
